scripts/utils/ttGen.ps1

```diff
@@ -0,0 +1,12 @@
+Write-Host "Regenerating all generated files from all .tt files."
+Write-Host "(requires t4.exe from 'dotnet tool install --global dotnet-t4' or right click .tt files in VS and 'run custom tool')"
```

Comment:
You can add a local tools manifest.
https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools#install-a-local-tool

Then it will work automatically.

Reply:
more automatically...yes...still needs a dotnet tool restore -- or should we add that tool restore into configure.ps1?

Reply:
yeah the idea would be to add it to configure.ps1.

Reply:
I've opened a tracking issue for that work: https://github.com/nuget/home/issues/9379